draft-ietf-avtcore-rtp-over-quic.md

```diff
@@ -411,42 +411,45 @@ extension, that can improve the precision of the feedback from QUIC is
 {{!I-D.draft-ietf-quic-ack-frequency}}, which allows a sender to control the
 delay of acknowledgments sent by the receiver.
 
+The list of RTCP Receiver Reports that could be replaced by feedback from QUIC follows:
+
 * *Receiver Reports* (`PT=201`, `Name=RR`, {{!RFC3550}})
-  * *Fraction lost*: The fraction of lost packets can be directly infered from
+  * *Fraction lost*: When RTP packets are carried in QUIC datagrams, the fraction of lost packets can be directly inferred from
     QUIC's acknowledgments. The calculation SHOULD include all packets up to the
     acknowledged RTP packet with the highest RTP sequence number. Later packets
     SHOULD be ignored, since they may still be in flight, unless other QUIC
-    packets that were sent after the datagram frame, were already acknowledged.
+    packets that were sent after the RTP packet frame, were already acknowledged.
   * *Cumulative lost*: Similar to the fraction of lost packets, the cumulative
-    loss can be infered from QUIC's acknowledgments including all packets up to
+    loss can be inferred from QUIC's acknowledgments including all packets up to
     the latest acknowledged packet.
   * *Highest Sequence Number received*: The highest sequence number received is
-    the sequence number of all RTP packets that were acknowledged.
-  * Interarrival jitter: If QUIC acknowledgments carry timestamps as described
+    the highest sequence number of all RTP packets carried in a QUIC packet that was acknowledged.
+  * *Interarrival jitter*: If QUIC acknowledgments carry timestamps as described
     in one of the extensions referenced above, senders can infer from QUIC acks
     the interarrival jitter from the arrival timestamps.
-  * Last SR: Similar to RTP arrival times, the arrival time of RTCP Sender
+  * *Last SR*: Similar to RTP arrival times, the arrival time of RTCP Sender
     Reports can be inferred from QUIC acknowledgments, if they include
     timestamps.
-  * Delay since last SR: This field is not required when the receiver reports
+  * *Delay since last SR*: This field is not required when the receiver reports
     are entirely replaced by QUIC feedback.
 * *Negative Acknowledgments* (`PT=205`, `FMT=1`, `Name=Generic NACK`,
   {{!RFC4585}})
   * The generic negative acknowledgment packet contains information about
     packets which the receiver considered lost. {{Section 6.2.1. of !RFC4585}}
     recommends to use this feature only, if the underlying protocol cannot
     provide similar feedback. QUIC does not provide negative acknowledgments,
-    but can detect lost packets through acknowledgments.
+    but can detect lost packets based on the Gap numbers contained in QUIC ACK frames.
```

Comment:
Should we also reference [Section 6 of RFC 9002](https://www.rfc-editor.org/rfc/rfc9002.html#section-6) here?
```suggestion
    but can detect lost packets based on the Gap numbers contained in QUIC ACK frames {{Section 6 of !RFC9002}}.
```